WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -172,7 +172,7 @@ public static async Task ShellExecAsync(string cmd, bool waitForExit = true)
 			using var process = Process.Start(
 				new ProcessStartInfo
 				{
-					FileName = "/usr/bin/env sh",
+					FileName = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? "/bin/sh" : "/usr/bin/env sh",
```

Comment:
@lontivero are you sure about Linux? I am thinking to revert this to the original one and remove complexity.

Reply:
https://github.com/zkSNACKs/WalletWasabi/issues/4224#issuecomment-679160573 I would try this first and if it is not a viable path, I would revert it.

Reply:
Great, thanks!